### DISCUSSION TITLE
Breaking chorus will drop existing flowers

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#999] [__Dismissed__] [answered]
Nobody seems to know how planting/replanting chorus fruit works. It would be nice to have the chorus flowers drop if you break the stem, instead of having to break each flower individually first.

**Comment by Keeg-Turner** [comment, marked as answer]
This is already a feature in [PurpurExtras](https://github.com/PurpurMC/PurpurExtras/wiki#chorus-flowers-always-drop).

**Reply by RhythmicSys** [reply]
Well
Yes
I made the feature in purpurExtras
I did forget to close this suggestion though xD
